lib/Onyx.js

```diff
@@ -561,6 +612,8 @@ function keyChanged(key, data, canUpdateSubscriber) {
  *     - sets state on the withOnyxInstances
  *     - triggers the callback function
  *
+ * Note: This is only invoked from Onyx.connect
```

Comment:
while true, I'd maybe remove this as comments like this can become of date and confuse people.